src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointPair.java

```diff
@@ -0,0 +1,476 @@
+        } else if (chimericAlignment.isNotSimpleTranslocation) { // no translocation
+
+            final int d1 = r2b - r1e - 1, // distance-1 between the two regions on reference
+                      d2 = c2b - c1e - 1; // distance-1 between the two regions on contig
+
+            String homologyForwardStrandRepresentation = EMPTY_STRING, insertedSeqForwardStrandRepresentation=EMPTY_STRING, dupSeqForwardStrandRepresentation=EMPTY_STRING;
+            int refDup=0, ctgDup=0;
+            // when d1<d2 (but not when both are negative), it always indicate an insertion, and vice versa
+            // when d2<0, there's always homologyForwardStrandRep
+            if (d1>0 && d2==0) {        // DEL: simple deletion, deleted sequence is [r1e+1, r2b-1] on the reference
+                homologyForwardStrandRepresentation    = EMPTY_STRING;
+                insertedSeqForwardStrandRepresentation = EMPTY_STRING;
+                dupSeqForwardStrandRepresentation      = EMPTY_STRING;
+                refDup = ctgDup = 0;
+            } else if (d1==0 && d2>0) { // INS: simple insertion, inserted sequence is the sequence [c1e+1, c2b-1] on the contig
+                homologyForwardStrandRepresentation    = EMPTY_STRING;
+                insertedSeqForwardStrandRepresentation = getInsertedSequence(firstContigRegion, secondContigRegion, contigSeq);
+                dupSeqForwardStrandRepresentation      = EMPTY_STRING;
+                refDup = ctgDup = 0;
+            } else if (d1==0 && d2<0) { // DEL: tandem duplication on reference (CN = 2) but one copy was deleted on the contig; duplicated sequence on reference are [r1e-|d2|+1, r1e] and [r2b, r2b+|d2|-1]
+                homologyForwardStrandRepresentation    = getHomology(firstContigRegion, secondContigRegion, contigSeq);
+                insertedSeqForwardStrandRepresentation = EMPTY_STRING;
+                dupSeqForwardStrandRepresentation      = homologyForwardStrandRepresentation;
+                refDup      = 2;
+                ctgDup      = 1;
+            } else if (d1<0 && d2==0) { // INS: tandem duplication of reference bases [r1e-|d1|+1, r1e] to contig bases [c1e-|d1|+1, c1e] and [c2b, c2b+|d1|-1]
+                final byte[] insertedSeq = Arrays.copyOfRange(contigSeq, c1e-Math.abs(d1), c1e);
+                if (!chimericAlignment.isForwardStrandRepresentation) SequenceUtil.reverseComplement(insertedSeq);
+                homologyForwardStrandRepresentation    = EMPTY_STRING;
+                insertedSeqForwardStrandRepresentation = EMPTY_STRING;
+                dupSeqForwardStrandRepresentation      = new String(insertedSeq);
+                refDup      = 1;
+                ctgDup      = 2;
+            } else if (d1<0 && d2>0) {  // INS: tandem duplication of reference bases [r1e-|d1|+1, r1e] to contig bases [c1e-|d1|+1, c1e] and [c2b, c2b+|d1|-1] and an inserted sequence [c1e+1, c2b-1]
```

Comment:
I just meant that the word 'tandem' as used in your comment doesn't really apply, I think, since there's an intervening inserted sequence if I'm understanding things correctly.